composer.lock

```diff
@@ -73,7 +73,7 @@
                 "docblock",
                 "parser"
             ],
-            "time": "2015-08-31 12:32:49"
+            "time": "2015-08-31T12:32:49+00:00"
```

Comment:
the issue with it that usually the user won't update it, which mean he will use outdated once